### DISCUSSION TITLE
Adding a extra NIC

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#2102] [Unsupported Versions] [answered]
Hello,

Is there a command to set the networks interfaces(Mgt & Mon interface) without doing a new installation, I want to add a Mon interface.

ISO standalone install upgraded to 2.3.10

Regards

Bart

**Comment by TOoSmOotH** [comment, marked as answer]
Salt doesn't manage the network interfaces so as long as you add the new interface to the bond that was created at setup, you should be good to go.

**Reply by sleepingbel** [reply]
Thx
I will try it.

Regards 
Bart
